cli/stork-publisher-agent/runner.go

```diff
@@ -10,16 +10,16 @@ import (
 )
 
 type PublisherAgentRunner[T Signature] struct {
-	config                  StorkPublisherAgentConfig
-	signatureType           SignatureType
-	logger                  zerolog.Logger
-	ValueUpdateCh           chan ValueUpdate
-	signedPriceBatchCh      chan SignedPriceUpdateBatch[T]
-	registryClient          *RegistryClient
-	brokerMap               map[BrokerPublishUrl]map[AssetId]struct{}
-	outgoingConnections     map[BrokerPublishUrl]*OutgoingWebsocketConnection[T]
-	outgoingConnectionsLock sync.RWMutex
-	signer                  Signer[T]
+	config                      StorkPublisherAgentConfig
+	signatureType               SignatureType
+	logger                      zerolog.Logger
+	ValueUpdateCh               chan ValueUpdate
+	signedPriceBatchCh          chan SignedPriceUpdateBatch[T]
+	registryClient              *RegistryClient
+	assetsByBroker              map[BrokerPublishUrl]map[AssetId]struct{}
+	outgoingConnectionsByBroker map[BrokerPublishUrl]*OutgoingWebsocketConnection[T]
```

Comment:
renamed for clarity - one map stores the connection object and the other stores the assets we want to send through that connection.